apps/reciptor-server/src/test/kotlin/de/mattagohni/reciptorserver/controller/ToolsControllerTest.kt

```diff
@@ -141,4 +141,40 @@ class ToolsControllerTest {
 
     verify(exactly = 1) { toolsService.saveTool(any()) }
   }
+
+  @Test
+  @DisplayName("it deletes an existing tool")
+  @WithMockUser
+  fun deleteTool_OK() {
+    // arrange
+    val toolToDelete = Tool(id = 4711, name = "knife")
+
+    every { toolsService.findToolById(4711) }.returns(Mono.just(toolToDelete))
+    every { toolsService.delete(4711) }.returns(Mono.just(4711))
+
+    // act
+    webTestClient.delete().uri("/api/v1/tools/4711")
+      .exchange()
+      .expectStatus().isEqualTo(HttpStatus.OK)
```

Comment:
Should we add an verification of the response body too? :)

Reply:
yeah, I will add an assertion for that.

Reply:
done!